eve/main.yml

```diff
@@ -2,7 +2,7 @@
 version: "0.2"
 
 branches:
-  default:
+  user/*, feature/*, improvement/*, bugfix/*, w/*, q/*, hotfix/*:
```

Comment:
A lot of user branches currently in this repo use the `users/*` pattern - if we merge this, would be nice to advertise the change to all developers